src/Featurizers/Components/FrequencyEstimator.h

```diff
@@ -0,0 +1,169 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#pragma once
+
+#include <limits>
+
+#include "TrainingOnlyEstimatorImpl.h"
+#include "GrainFeaturizerImpl.h"
+
+namespace Microsoft {
+namespace Featurizer {
+namespace Featurizers {
+namespace Components {
+
+static constexpr char const * const         FrequencyEstimatorName("FrequencyEstimator");
+static constexpr char const * const         GrainFrequencyEstimatorName("GrainFrequencyEstimator");

```

Comment:
I don't believe that this is necessary anymore.

Reply:
We are using it on line 178 in the `GlobalMinimumFrequencyEstimator.h` to get the annotations. We could just use it there and not define it, but we were thinking that if we ever want to get these annotations anywhere else that it would make sense to have it defined somewhere.

Reply:
Per offline discussion, merging for now and will create a new PR with these changes.